third_party/proto/tendermint/crypto/keys.proto

```diff
@@ -11,6 +11,7 @@ message PublicKey {
   option (gogoproto.equal)   = true;
 
   oneof sum {
-    bytes ed25519 = 1;
+    bytes ed25519   = 1;
+    bytes secp256k1 = 2;
```

Comment:
@amaurymartiny not sure if this is relevant to you as you look into bringing secp256k1 support back into the sdk. Just flagging here incase its useful.

Reply:
Got it, thanks 👍